src/sources/forus-webshop/pug/tpl/directives/top-navbar.pug

```diff
@@ -1,6 +1,6 @@
 .block.block-navbar: .navbar-inner.wrapper
     a(ui-sref="home" ng-if="$root.appConfigs.client_key != 'general'").navbar-logo
-        img(ng-src="{{ './assets/img/SDOA.svg' }}" ng-if="$root.appConfigs.flags.SDOA").second-logo
+        img(ng-src="{{ './assets/img/sdoa-logo.svg' }}" ng-if="$root.appConfigs.flags.showSdoaLogo").second-logo
```

Comment:
I would be okay with changing the name of this variable to showSecondLogo and also renaming the svg. As in the future it could be used for a different logo. (not only SDOA)